src/styles/renderIndividual.module.css

```diff
@@ -18,7 +18,56 @@
   display: flex;
   justify-content: center;
   align-items: center;
-  gap: 20px;
+  gap: 10px;
+}
+
+.renderWrapperCurrentStateShape {
+  position: relative;
+  width: 5rem;
+  height: 5rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.rectangle3 {
+  width: 4rem;
+  height: 4rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.rectangle3 svg {
+  position: relative;
+  width: 70px;
+  height: 70px;
+}
+
+.rectangle3 svg circle {
+  width: 150px;
+  height: 150px;
+  fill: none;
+  stroke-width: 5;
+  stroke: #000;
+  transform: translate(5px, 5px);
+  -webkit-transform: translate(5px, 5px);
+  -moz-transform: translate(5px, 5px);
+  -ms-transform: translate(5px, 5px);
+  -o-transform: translate(5px, 5px);
+  stroke-dasharray: 440;
+  stroke-dashoffset: 440;
+}
+
+.rectangle3 svg circle:nth-child(1) {
+  stroke-dashoffset: 0;
+  stroke: #03a9f4;
+}
+
+.rectangle3 svg circle:nth-child(2) {
+  stroke-dashoffset: -99;
+  stroke: #f3f3f3;
 }
 
 .renderWrapperCurrentChapter {
```

Comment:
As required for the project, your design to stick as much as possible to the one provided on [Zeplin](https://zeplin.io/) and All elements in design to be added. kindly ensure that you improve your Header styling by using the correct fonts(size, weight, family,...) on text, correct alignment, padding, margin, text color, height, background color on buttons, spacing between book item, height of book item,... Just see the below images to see the difference
#### Template design
![Screenshot from 2023-05-19 18-48-53](https://github.com/Elhambasir/bookstore-add-reducers-and-actions/assets/83042742/79ef7f0d-b838-4ace-b746-9e0d4f8e061d)
#### Your design
![Screenshot from 2023-05-19 18-47-56](https://github.com/Elhambasir/bookstore-add-reducers-and-actions/assets/83042742/dce304e4-7a7c-48e4-ac44-b73791b98999)

Reply:
Ok thank you